README.md

```diff
@@ -501,52 +501,41 @@ There are a number of programs that come with PIH EMR. Each one has its own comp
-
-Steps to add a messages_es.properties:
-1) Create the messages_es.properties file in the resources directory of the api project of the module:
-[https://github.com/PIH/openmrs-module-mirebalais/tree/master/api/src/main/resources](https://github.com/PIH/openmrs-module-mirebalais/tree/master/api/src/main/resources)
-
-
-2) Add the locale "es" to the config.xml of the module:
-[https://github.com/PIH/openmrs-module-mirebalais/blob/master/omod/src/main/resources/config.xml#L138](https://github.com/PIH/openmrs-module-mirebalais/blob/master/omod/src/main/resources/config.xml#L138)
-
-
-3) Add the locale "es" to the Transifex configuration file for the module:
-[https://github.com/PIH/openmrs-module-mirebalais/blob/master/.tx/config](https://github.com/PIH/openmrs-module-mirebalais/blob/master/.tx/config)
-
-Of course, after we do this, we need to start having translators edit translations and pull them into the projects.
-
-Documentation can be found here:
-https://wiki.openmrs.org/display/docs/Maintaining+OpenMRS+Module+Translations+via+Transifex
-(In particular see the section 'Updating A Module With New Translations" and how to install the Transifex command line client).
+See the [OpenMRS Wiki page on locales](https://wiki.openmrs.org/display/docs/Localization+and+Languages).
+
+Allowed locales are configured in [global properties](https://github.com/PIH/openmrs-config-pihliberia/blob/880704d98f477ec75db9c29bc5566ac9c90a5aad/configuration/globalproperties/gp_pihliberia.xml#L10).
+
+Text mostly comes either from concept names or from message strings.
+	
+Concept names and their translations should be managed on the [concepts server](concepts.pih-emr.org).
+
+### Message string management
+	
+Some message strings are embedded into OpenMRS modules. See, for example, the
+[English](https://github.com/openmrs/openmrs-module-coreapps/blob/master/api/src/main/resources/messages.properties)
+and [Spanish](https://github.com/openmrs/openmrs-module-coreapps/blob/master/api/src/main/resources/messages_es.properties)
+strings in the coreapps module.
+	
+Message strings that are specific to the PIH EMR are kept in [config-pihemr](https://github.com/PIH/openmrs-config-pihemr/tree/master/configuration/messageproperties).
+	
+In either case, those strings are managed through [Transifex](https://www.transifex.com/pih/mirebalais/dashboard/).
+Ask someone from the PIH EMR team to grant you access to the PIH Transifex org.
+Those strings will be downloaded and used to update the `messages_**.properties` files.
+*Do not* attempt to translate the EMR by modifying those files directly.
+
+### Transifex Workflow
+
+1. Navigate to the PIH org in Transifex
+1. Click "Resources"
+1. Click on the resource you want to work on. The messages in [config-pihemr](https://github.com/PIH/openmrs-config-pihemr/tree/master/configuration/messageproperties)
```

Comment:
Looks like the editor formatted this as a bunch of "1." steps...?

Reply:
It's a markdown feature; makes it so you can re-order and insert and remove steps without having to re-number all the others. https://guides.github.com/features/mastering-markdown/#syntax